.github/tailor.yaml

```diff
@@ -0,0 +1,10 @@
+rules:
+  - name: commit title
+    description: all commit titles must have a scope, be lowercase, and be no more than 50 characters
+    expression:  |-
+      .commits all(((.message test "^[a-z/\\-\\*]+: [[:alnum:] \\-'./]+") and (.message test "^.{0,50}\\n")) or (.message test "^Revert"))
```

Comment:
I don't think `*` needs to be escaped in character classes, and the traditional way to deal with `-` is to put it at the end.

We will probably discover additional useful punctuation over time; I, personally, enjoy semicolons.  But this seems like a good starting point.

Reply:
Updated with your suggestions.